sdk/sdkservices/runtimes.go

```diff
@@ -46,22 +47,31 @@ type Runtime interface {
 }
 
 type (
-	RunLoadFunc  = func(ctx context.Context, rid sdktypes.RunID, path string) (map[string]sdktypes.Value, error)
-	RunCallFunc  = func(ctx context.Context, rid sdktypes.RunID, v sdktypes.Value, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error)
-	RunPrintFunc = func(ctx context.Context, rid sdktypes.RunID, text string)
-	NewRunIDFunc = func() sdktypes.RunID
+	RunLoadFunc            = func(ctx context.Context, rid sdktypes.RunID, path string) (map[string]sdktypes.Value, error)
```

Comment:
Why do we need these types, you can specify the function signature in `RunCallbacks` and that's it.

Reply:
removed.